envs/example/defaults.yml

```diff
@@ -40,6 +40,11 @@ memcached:
   user: nobody
   max_connections: 1024
 
+swift:
+  enabled: False
+  haproxy_vip: "{{ fqdn }}"
```

Comment:
The swift end point gets its own DNS name and VIP, so that object traffic doesn't conflict with other cloud API traffic. So setting this to fqdn seems wrong.